app/views/secrets/show.html.erb

```diff
@@ -69,6 +69,9 @@
           <%= submit_tag ResourceController::ADD_MORE, class: 'btn btn-default' %>
           <% if id %>
             <%= link_to "Duplicates", secrets_path(search: {value_from: id}) %>
+            <% if Samson::Secrets::Manager.backend == Samson::Secrets::HashicorpVaultBackend %>
```

Comment:
Need to check for vault v2 / history support?

Reply:
killing that soon, so not adding extra steps now